Sources/Fluent/Pivot/PivotProtocol.swift

```diff
@@ -59,8 +59,11 @@ extension PivotProtocol where Self: Entity {
         let rightId = try right.assertExists()
 
         try makeQuery()
-            .filter(Left.foreignIdKey, leftId)
-            .filter(Right.foreignIdKey, rightId)
+            .filter(pivotLeftIdKey, leftId)
+            .filter(pivotRightIdKey, rightId)
             .delete()
     }
 }
+
+public var pivotLeftIdKey: String = "left_id"
```

Comment:
That would break the siblings relationship. It relies on being able to do joins across both pivot tables and non-pivot tables alike using the entity's foreign key.